packages/babel-plugin-transform-react-jsx-development/test/fixtures/windows/classic-runtime-with-source-self-plugins-windows/options.json

```diff
@@ -1,4 +1,5 @@
 {
+  "skip": true,
```

Comment:
If this doesn't work (I have never seen this used before), you'll need to rename the folder to `.classic-runtime-with-source-self-plugins-windows` (with `.`) instead.

Reply:
Actually, instead of disabling this test we could remove `...props` from the fixture input, and make it only throw in Babel 8?

Reply:
It didn't :trollface: was worth trying!